affected/package: trpc alias not work

<!--
Please answer these questions before submitting your issue. Thanks!
-->

### What version of trpc-cmdline are you using?
trpc-group/trpc-cmdline version: v1.0.5

### Does this issue reproduce with the latest release?
yes 

### What operating system and processor architecture are you using (`go env`)?
CentOS 7.9
<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
GOVERSION="go1.20.10"

</pre></details>

### What did you do?
i want create *go file and swagger.json file  from proto file use by  trpc-cmdline . use alias tag 

helloword.proto

```
// HelloWorldService handles hello request and echo message.
service HelloWorldService {
  // Hello says hello.
  rpc Hello(HelloRequest) returns(HelloResponse);

  // Apply
  //@alias="/api/apply"
  rpc Apply(ApplyRequest) returns(ApplyResponse){};
}

```

use trpc command line 

`trpc create -p ./helloworld.proto -o helloworld --rpconly --swagger --alias`

> [create] run goimports -w . error: exit status 2,
> helloworld.trpc.go:89:40: missing ',' in composite literal
> helloworld.trpc.go:103:5: missing ',' in composite literal
> helloworld.trpc.go:104:71: missing ',' before newline in composite literal
> helloworld.trpc.go:105:6: missing ',' before newline in composite literal
> helloworld.trpc.go:106:2: missing ',' before newline in composite literal
> helloworld.trpc.go:112:1: expected operand, found 'type'
> helloworld.trpc.go:116:52: missing ',' in argument list
> helloworld.trpc.go:117:5: missing ',' in argument list
> helloworld.trpc.go:118:1: expected operand, found '}'
> helloworld.trpc.go:121:5: missing ',' in argument list
> helloworld.trpc.go:122:1: expected operand, found '}'
> 
> Execution err:
>         running plugin `goimports`, err: exit status 2
> Please run "trpc -h" or "trpc create -h" (or "trpc {some-other-subcommand} -h") for help messages.
> 

in  helloworld.trpc.go ,it show error 

```
            {
                Name: ""/api/apply"",
                Func: HelloWorldServiceService_Apply_Handler,
                
            },
```
double ' " '  made it error .  


if i cancel  `--alias` arg, it works good . 
but i want  use  'alias' tag define my own url , so  what should i do now？ 



<!--
If possible, provide a recipe for reproducing the error.
A complete runnable program is good.
A link on go.dev/play is best.
-->


### What did you expect to see?

alias tag work well . 
in that case , in  swagger file ,this api  
```
path = /api/apply
summary = Apply
```



### What did you see instead?

--- a/parser/fill.go
+++ b/parser/fill.go
@@ -765,7 +765,7 @@ func isCommentDiff(leadingComment string, leadingErr error, trailingComment stri
 	return (leadingErr == nil && trailingErr == nil) && (leadingComment != trailingComment)
 }
 
-func parseAlias(comment string) (string, error) {
+func parseAlias(comment string) (alias string, err error) {
 	const marker = "@alias="
 	if !strings.Contains(comment, marker) {
 		return "", fmt.Errorf("annotation alias %s not found in raw comment %s", marker, comment)
@@ -784,7 +784,10 @@ func parseAlias(comment string) (string, error) {
 		return "", fmt.Errorf("candidate alias %s is double commented", s[0])
 	}
 
-	alias := strings.TrimSpace(s[1])
+	defer func() {
+		alias = strings.Trim(alias, " \"'")
+	}()
+	alias = strings.TrimSpace(s[1])
 	if len(alias) == 0 {
 		return "", fmt.Errorf("invalid alias after trim space: %s", comment)
 	}

